ucca/core.py

```diff
@@ -400,10 +472,10 @@ def __repr__(self):
         return Node.__name__ + "(" + self.ID + ")"
 
     @ModifyPassage
-    def add(self, edge_tag, node, *, edge_attrib=None):
+    def add(self, edge_categories, node, *, edge_attrib=None):
         """Adds another :class:`Node` object as a child of self.
 
-        :param edge_tag: the label of the :class:`Edge` connecting between the
+        :param edge_categories: the labels of the :class:`Edge` connecting between the
```

Comment:
Need more informative documentation on the expected type of the list elements (4-tuples) if we keep it this way